migtests/tests/ora-foreign-key-tests/env.sh

```diff
@@ -0,0 +1,13 @@
+export SOURCE_DB_TYPE="oracle"
+export SOURCE_DB_NAME=${SOURCE_DB_NAME:-"ORCL"}
+export SOURCE_DB_PORT=${SOURCE_DB_PORT:-1521}
+export SOURCE_DB_USER="ADMIN"
+export SOURCE_DB_PASSWORD="Password1"
+export SOURCE_DB_HOST="cnaidu-oracle-rds.cbtcvpszcgdq.us-west-2.rds.amazonaws.com"
+export SOURCE_DB_SCHEMA="ADMIN"
+export TARGET_DB_HOST="10.9.200.131"
+export TARGET_DB_PORT="5433"
+export TARGET_DB_USER="ybvoyager"
+export TARGET_DB_PASSWORD="password"
+export TARGET_DB_NAME="migration_test"
+export TARGET_DB_SCHEMA="non_public_test"
```

Comment:
This env.sh file is only meant to contain SOURCE_DB_TYPE, SOURCE_DB_NAME, and SOURCE_DB_SCHEMA.
Please, define the other variables in your shell environment.